cvat-ui/src/containers/annotation-page/top-bar/top-bar.tsx

```diff
@@ -672,6 +699,8 @@ class AnnotationTopBarContainer extends React.PureComponent<Props, State> {
                     undoShortcut={normalizedKeyMap.UNDO}
                     redoShortcut={normalizedKeyMap.REDO}
                     drawShortcut={normalizedKeyMap.SWITCH_DRAW_MODE}
+                    // this shortcut is handled in interactionHandler.ts separatelly
+                    blockShortcut='Ctrl'
```

Comment:
But we can add it to general shortcuts list to show it in F2 with an appropriate description

Reply:
Also let's rename it to ``switchToolsBlockerShortcut``

Reply:
I've added this shortcut to defaultKeyMap with description